### PR TITLE
Issue 3270 - CONTRIBUTING.md fixes and readability enhancements

### LINKED ISSUE #3270: Fix typos and mistakes in the Contributions Guidelines
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.

### What would you like to improve?

These are Ivy Wallet's contribution guidelines:
https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md

Fix typos, mistakes and in general make them easier to understand and follow.

### Because...

- make the contribution process easier
- motivate more developers to contribute

### Description

_No response_

### Success Criteria

- The guidelines are easier to read after your changes.
- The guidelines are easier to understand. 
- Grammar/typos is improved.

### PR DESCRIPTION
**Pull request by CodinGeo**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [✅] The PR is submitted to the `main` branch.
- [✅] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [✅] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [✅] I confirm that I've run the code locally and everything works as expected.
- [✅] 🎬 I've attached a **screen recoding** of the changes.
- [✅] My PR includes only the necessary changes to fix the issue (i.e., no unnecessary files or lines of code are changed).

## What's changed?
Before:
image
image
image
image

After:
image
image
image

Describe with a few bullets **what's new:**
- Texts were made simpler and easier to read
- I've deleted some of the more graphic things (like forking badge) in order to make the ⚠️ icons more visible.
- I've changed place of some info in order to keep them in one place based on their topic
- Forking part was moved after finding an issue

## Risk Factors

**What may go wrong if we merge your PR?**

- Some people may not like the design, but I think the content is now much easier to follow.

**In what cases your code won't work?**

-  I don't see any, it's a .md file for developers ;)

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #3270

## Troubleshooting CI failures ❌

GitHub Actions failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]
Hey @CodinGeo let's:
1. Move forking as first and bring back the graphic badge
- Forks leads to repo SEO, popularity and stats
- Once a person forks they're more likely to contribute

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,87 +1,67 @@
 # Contributing to Ivy Wallet
 
-## 1. Fork the repo
 
-1. Install the git Large File Storage plugin, see [here](https://git-lfs.com/) for instructions.
-2. Create a fork by clicking this badge: 👉 [![Fork Ivy Wallet](https://img.shields.io/github/forks/Ivy-Apps/ivy-wallet?logo=github&style=social)](https://github.com/Ivy-Apps/ivy-wallet/fork) 👈
```
Comment (ILIYANGERMANOV) [inline CONTRIBUTING.md:6]: Bring the Fork badge back

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,87 +1,67 @@
...
-
-**[How To Fork Guide by GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo)**
-
-## 2. Pick an issue
-
-What do you want to work on? [Choose an issue](https://github.com/Ivy-Apps/ivy-wallet/issues) that you'd like to contribute to.
+## 1. Pick an issue
```
Comment (ILIYANGERMANOV) [inline CONTRIBUTING.md:4]: Make forking step one

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,8 +1,13 @@
 # Contributing to Ivy Wallet
 
 
-## 1. Pick an issue
+## 1. Fork the repo
+Fork of the official Ivy Wallet repo by clicking on the badge -> [![Fork Ivy Wallet](https://img.shields.io/github/forks/Ivy-Apps/ivy-wallet?logo=github&style=social)](https://github.com/Ivy-Apps/ivy-wallet/fork).
```
Comment (ILIYANGERMANOV) [inline CONTRIBUTING.md:5]: Change the "->" to ":" so it looks more anesthetic

**Review by ILIYANGERMANOV — APPROVED** [review]
Amazing work 👏

**Comment by CodinGeo** [comment]
@ILIYANGERMANOV There is no other way to change such simple things like this arrow without just commiting, right?

**Comment by ILIYANGERMANOV** [comment]
> @ILIYANGERMANOV There is no other way to change such simple things like this arrow without just commiting, right?

Each change, even 1 symbol requires a commit. GIT uses an immutable history where each change is a commit (similarly to blockchain)

**Comment by ILIYANGERMANOV** [comment]
@CodinGeo do you want to have a stab and make the PR template more user friendly, as well? If so feel free to create an issue and I'll approve it

**Pull request merged**